src/utils/validation.ts

```diff
@@ -125,14 +123,7 @@ const loadPhoneValidator = () =>
 
 const validators = {
   email: (a: string) => emailPattern.test(a),
-  phone: (a: string) => {
-    try {
-      return global.libphonenumber.isValidPhoneNumber(a, 'US');
-    } catch (e) {
-      // Invalid phone number
-      return false;
-    }
-  },
+  // should this be changed to just phone, now that it is the only phone validator?
   internationalPhone: (a: string) => {
```

Comment:
change this to `phone`  since it is now the only used phone validator?

Reply:
yep agreed